src/backend/distributed/executor/multi_utility.c

```diff
@@ -1595,19 +1573,15 @@ ReplicateGrantStmt(Node *parsetree)
 	isFirst = true;
 	foreach(granteeCell, grantStmt->grantees)
 	{
-#if (PG_VERSION_NUM >= 90500)
 		RoleSpec *spec = lfirst(granteeCell);
-#else
-		PrivGrantee *spec = lfirst(granteeCell);
-#endif
 
 		if (!isFirst)
 		{
 			appendStringInfoString(&granteesString, ", ");
 		}
+
```

Comment:
that was to follow "empty line after the code block" convention. But is it not related to 9.4 here.